drivers/usb/device/usb_dc_stm32.c

```diff
@@ -350,7 +357,7 @@ static int usb_dc_stm32_init(void)
 	usb_dc_stm32_state.pcd.Init.ep0_mps = PCD_EP0MPS_64;
 	usb_dc_stm32_state.pcd.Init.low_power_enable = 0;
 #else /* USB_OTG_FS || USB_OTG_HS */
-#if DT_HAS_COMPAT_STATUS_OKAY(st_stm32_otghs)
+#if DT_HAS_COMPAT_STATUS_OKAY(st_stm32_otghs) && !defined(CONFIG_SOC_SERIES_STM32H7X)
```

Comment:
Please check if still required after fixing compatible to otgfs

Reply:
This was no longer necessary when using `otgfs`, removed.